### ISSUE TITLE
SetUp Issues & Doubts

### ISSUE THREAD

**Issue by NoobInTheNet** [moloch--/RootTheBox#214]
Hi everyone! 

After solving some compatibility problems when starting "RootTheBox" I got it started, but it keeps **"Starting application ..."** and I can't get the process forward.

**Comment by jeffg2k** [comment]
Make sure your using Python 2.7 and not Python 3.  RTB is not yet supported on Python 3.  Otherwise, I'll need more information.  Maybe put it in debug mode.

**Comment by NoobInTheNet** [comment]
Next I attach the resulting log when executing the .py in debug mode:

[rtb.log](https://github.com/moloch--/RootTheBox/files/2776645/rtb.log)

**Comment by jeffg2k** [comment]
Yes, "Starting application" is the last statement at the command line indicating that the web service is started.  You will get additional information logged at the command line when you start the game or if you have it in debug mode, you'll see the connections made to the web server

Looks like you're running it on port 8888, so you should be able to connect to it at `http://localhost:8888` for that machine or based on your origin, at `http://192.168.1.1:8888`.  If you're trying to access it externally, then you'll want to put the public IP in the origin and if you want Google to index it for search, you probably want to put it on port 80 or get a cert, like Let's Encrypt, and put it on port 443.

**Comment by NoobInTheNet** [comment]
Cool! Everything works, now I will try to customize it and translate it into Spanish.

Thank you very much!

**Comment by NoobInTheNet** [comment]
I've already installed a SSL certificate and all works good.

But now, I have a problem, I dont know the "path" to acces the admin interface.

**Comment by jeffg2k** [comment]
If you're speaking of the admin interface in the site itself, then you'd just log in with admin or whatever you set as the admin account.  RTB will display the admin menus if you're an admin.

If you're talking about "on disk", it sort of depends what aspect you're speaking about as it's not one component.  You have the backend handlers for admin, which is in /handlers/AdminHandlers. You have the html templates, which is in /templates/admin.  You have the javascript, which is in /static/js/pages/admin, for example.

**Comment by NoobInTheNet** [comment]
Okay, all solved, I forgot that I noted a user and pass when I first started RTB.

Thankyou very much. ;D
